src/java/mars/rockets/U1.java

```diff
@@ -4,15 +4,27 @@
 
 public class U1 extends Rocket {
 
+    public U1(double cost, double weight, double cargo) {
+        super(cost, weight, cargo);
+    }
+
     @Override
     public boolean launch(){
-        // implement code here
-        return false;
+        return getProbabilityExplotion(0.05);
     }
 
     @Override
     public boolean land() {
-        // implement code here
-        return false;
+        return getProbabilityExplotion(0.01);
+    }
+
+    private boolean getProbabilityExplotion(double probability){
```

Comment:
this same method exists in U2 and the code is the same, would not it be better to implement it in the parent class as protected?